src/util/kaldi-table-inl.h

```diff
@@ -38,6 +40,25 @@ namespace kaldi {
 /// \addtogroup table_impl_types
 /// @{
 
+// In SequentialTableReaderScriptImpl and RandomAccessTableReaderScriptImple,
+// We use this function to separate the input string "line"
+// into the range specifier (e.g. [1:2,2:10]) and the data_rxfilename
+// (e.g. 1.ark:100). It returns true if successful.
+static bool ExtractRangeSpecifier(const std::string &line, 
```

Comment:
you shouldn't say 'static' in a header.
Probably best to just declare it here, and define it in kaldi-table.cc.